examples/service-clients/azure-client/external-controller/tests/index.ts

```diff
@@ -154,26 +146,34 @@ export async function createContainerAndRenderInElement(
 /**
  * For local testing we have two div's that we are rendering into independently.
  */
-async function setup() {
+async function setup(): Promise<void> {
```

Comment:
Which eslint config is this package using? If `minimal-deprecated`, this might be a good time to update it to `recommended` or `strict`. Either of those would enforce the presence of return types.

Reply:
There were a lot of complaints in VS Code. I was just getting the worst.